cdk/lib/gatehouse.ts

```diff
@@ -222,6 +222,7 @@ export class Gatehouse extends GuStack {
 			roleConfiguration: {
 				additionalPolicies: [readAppSsmParamsPolicy, xrayTelemetryPolicy],
 			},
+			instanceMetricGranularity: '5Minute',
```

Comment:
`5Minute` switches off enhanced monitoring which is the same as what we had before the upgrade.